graphyte.py

```diff
@@ -33,7 +33,7 @@ def _has_whitespace(value):
 class Sender:
     def __init__(self, host, port=2003, prefix=None, timeout=5, interval=None,
                  queue_size=None, log_sends=False, protocol='tcp',
-                 batch_size=1000, tags={}):
+                 batch_size=1000, raise_send_errors=False, tags=None):
```

Comment:
Regarding tags defaulting to `{}`: I didn't see why `tags={}` was incorrect here, as we're never modifying it. What's your thinking there? Either way, I don't think we should stuff it into this PR.

Nit: I know it shouldn't matter as these should be specified as keyword args, but could you add the new arg at the end?

Reply:
Honestly, not much thinking went into it.  I did it a bunch when I started out and was told never to do it that way.   I always change it in my old code when I find out.   It's my first time contributing to someone's project, should have left it alone!

Reply:
No worries, and thanks!